pyproject.toml

```
@@ -28,15 +28,15 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
-nibabel = "^2.3.3"
+nibabel = "^3.2.2"
 nipype = ">=1.7.1,<1.8.3"
 argcomplete = "^1.9.4"
 pandas = "^1.4"
 jinja2 = "^3"
 xvfbwrapper = "*"
 numpy = "^1.17"
 scikit-learn = "^1.0"
-nilearn = {version = "^0.7.0", extras = ["plotting"]}
+nilearn = {version = "^0.9.0", extras = ["plotting"]}
```

The `plotting` extras have been officially [introduced](https://github.com/nilearn/nilearn/commit/ae1fe0ab68d2e88a5957ba7ab610cb742f6aebbc) for version 0.9.1.

```suggestion
nilearn = {version = "^0.9.1", extras = ["plotting"]}
```